be/src/agent/task_worker_pool.h

```diff
@@ -51,12 +51,12 @@ class TaskWorkerPool {
         REALTIME_PUSH,
         PUBLISH_VERSION,
         // Deprecated
-        CLEAR_ALTER_TASK,
+        CLEAR_ALTER_TASK [[deprecated]],
```

Comment:
Just remove it?

Reply:
remove directly may change the enum value